core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -858,6 +860,33 @@ protected function switchToEdit($id)
 		return $strUrl . (!empty($arrKeys) ? '&' : '') . (Input::get('table') ? 'table=' . Input::get('table') . '&amp;' : '') . 'act=edit&amp;id=' . rawurlencode($id);
 	}
 
+	/**
+	 * @throws AccessDeniedException
+	 */
+	protected function denyAccessUnlessGranted($attributes, $subject): void
```

Comment:
```suggestion
	protected function throwUnlessGranted($attributes, $subject): void
```

Might make it easier to read the code that calls this method?

Reply:
`denyAccessUnlessGranted` is the method name Symfony already uses and is widely known and maybe even documented. I would stick with that.